src/sidebar/store/modules/selection.js

```diff
@@ -337,16 +337,19 @@ function selectedTab(state) {
 /**
  * @return {SelectionState}
  */
-function selectionState(state) {
-  const selectionState = {
-    expanded: expandedMap(state),
-    forcedVisible: forcedVisibleAnnotations(state),
-    selected: selectedAnnotations(state),
-    sortKey: sortKey(state),
-    selectedTab: selectedTab(state),
-  };
-  return selectionState;
-}
+const selectionState = createSelector(
+  state => state,
+  selection => {
+    const selectionState = {
+      expanded: expandedMap(selection),
+      forcedVisible: forcedVisibleAnnotations(selection),
+      selected: selectedAnnotations(selection),
+      sortKey: sortKey(selection),
+      selectedTab: selectedTab(selection),
+    };
+    return selectionState;
```

Comment:
This is fine, although I would probably have omitted the `selectionState` variable here.

Reply:
Done.